src/msg/async/rdma/RDMAStack.cc

```diff
@@ -245,7 +247,8 @@ void RDMADispatcher::polling()
       if (!num_qp_conn && done)
         break;
 
-      if ((ceph_clock_now() - last_inactive).to_nsec() / 1000 > cct->_conf->ms_async_rdma_polling_us) {
+      uint64_t now = Cycles::rdtsc();
+      if (Cycles::to_microseconds(now - last_inactive) > cct->_conf->ms_async_rdma_polling_us) {
```

Comment:
Use Cycles::to_nanoseconds() and use pre-calculated value of cct->_conf->ms_async_rdma_polling_us * 1000 to get rid of one floating point division, or better yet, precompute value of cct->_conf->ms_async_rdma_polling_us in cycles and compare without using Cycles::to_microseconds at all.
In any case, using RDTSC is prone to errors resulting from CPU frequency scaling and differing TSC values when thread switches between CPU cores. Not sure if reducing CPU usage (from 28% to 22%) is really worth the risk.